CONTRIBUTING.adoc

```diff
@@ -207,6 +207,8 @@ use cases.
 To run a dind cluster in a VM, follow steps 1-3 of the Vagrant
 instructions and then execute the following:
 
+        # Extra memory is required to build the extended tests
+        $ export OPENSHIFT_MEMORY=3072
```

Comment:
I think non-dind needs to be fixed separately.  Using 3gb for a single dind node is probably fine, but in a vm dev cluster only the master performs builds that require 3gb.  The nodes would be fine with 2gb, so it probably makes sense to allow separate configuration of master and node memory.